tests/containers/podman_docker_interop.pm

```diff
@@ -0,0 +1,125 @@
+# SUSE's openQA tests
+#
+# Copyright 2022-2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate docker and podman network interoperability
+# - validate if podman and docker networking works together
+# - check if firewalld doesn't break either
+# Maintainer: QE-C team <qa-c@suse.de>
+
+#use Mojo::Base qw(consoletest);
+use Mojo::Base qw(containers::basetest);
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+use version_utils qw(check_os_release get_os_release is_sle is_sle_micro is_transactional);
+use transactional qw(check_reboot_changes);
+#use containers::common;
+
+my $test_img = "registry.opensuse.org/opensuse/nginx";
+my $test_cmd = "curl https://opensuse.org/ -vso/dev/null";
+
+sub run {
+    my ($self, $args) = @_;
+
+    select_serial_terminal;
+
+    # Initialize and cleanup podman
+    my $podman = $self->containers_factory('podman');
+    $podman->cleanup_system_host();
+
+    assert_script_run("podman run --rm $test_img $test_cmd");
```

Comment:
This line is required, otherwise any subsequent outgoing podman container connections fail.
[https://openqa.opensuse.org/tests/4683681#step/podman_docker_interop/224](https://openqa.opensuse.org/tests/4683681#step/podman_docker_interop/224)
Any idea what might be causing this?

Reply:
Is it possible that I'm doing something wrong with OpenQA or is this a bug that should be submitted to Bugzilla?

Reply:
No no, I mean: This GitHub comment should also be in the code:
```suggestion
    # This line is required, otherwise any subsequent outgoing podman container connections fail.
    assert_script_run("podman run --rm $test_img $test_cmd");
```

Reply:
Done.